msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/msalonly/update/TestCaseUpdateAuthenticator.kt

```diff
@@ -0,0 +1,123 @@
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.msalonly.update
+
+import com.microsoft.identity.client.Prompt
+import com.microsoft.identity.client.msal.automationapp.R
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalAuthResult
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalAuthTestParams
+import com.microsoft.identity.client.msal.automationapp.sdk.MsalSdk
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalCustomBrokerInstallationTest
+import com.microsoft.identity.client.ui.automation.TokenRequestTimeout
+import com.microsoft.identity.client.ui.automation.annotations.RetryOnFailure
+import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator
+import com.microsoft.identity.client.ui.automation.constants.AuthScheme
+import com.microsoft.identity.client.ui.automation.interaction.PromptHandlerParameters
+import com.microsoft.identity.client.ui.automation.interaction.PromptParameter
+import com.microsoft.identity.client.ui.automation.interaction.microsoftsts.AadPromptHandler
+import com.microsoft.identity.labapi.utilities.client.LabQuery
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment
+import com.microsoft.identity.labapi.utilities.constants.TempUserType
+import org.junit.Test
+import java.util.*
+
+@RetryOnFailure
+class TestCaseUpdateAuthenticator : AbstractMsalCustomBrokerInstallationTest() {
+
+    private val mAuthenticator: BrokerMicrosoftAuthenticator = installOldAuthenticator()
+
+    @Test
+    @Throws(Throwable::class)
+    fun test_UpdateAuthenticator() {
+        val username = mLabAccount.username
+        val password = mLabAccount.password
+
+        val msalSdk = MsalSdk()
+        val authTestParams = MsalAuthTestParams.builder()
+            .activity(mActivity)
+            .loginHint(username)
+            .scopes(Arrays.asList(*mScopes))
+            .promptParameter(Prompt.LOGIN)
+            .authScheme(AuthScheme.POP)
+            .msalConfigResourceId(configFileResourceId)
+            .build()
+
+        val authResult = msalSdk.acquireTokenInteractive(authTestParams, {
+            val promptHandlerParameters = PromptHandlerParameters.builder()
+                .prompt(PromptParameter.LOGIN)
+                .loginHint(username)
+                .sessionExpected(false)
+                .consentPageExpected(false)
+                .speedBumpExpected(false)
+                .broker(mAuthenticator)
+                .expectingBrokerAccountChooserActivity(false)
+                .build()
+            AadPromptHandler(promptHandlerParameters)
+                .handlePrompt(username, password)
+        }, TokenRequestTimeout.MEDIUM)
+
+        // Check if auth result is success
+        authResult.assertSuccess()
+        MsalAuthResult.verifyATForPop(authResult.accessToken)
```

Comment:
we do not have to test PoP scenarios. Please remove this part. Changing the authScheme to bearer should also be part of the change you do remove PoP

Reply:
Updated code per suggestion.